src/structures/GuildBan.js

```diff
@@ -0,0 +1,35 @@
+const Base = require('./Base');
+
+/**
+ * Represent a Ban
+ */
+class GuildBan extends Base {
```

Comment:
This should probably have a fetch function of its own, due to it being "partial" when added from a GUILD_BAN_ADD event.

But at that rate you could also add a `remove` function for utility 
```js
guild.bans.fetch()
	.then(bans => {
		return bans.get('USER ID').remove();
	})
	.catch(console.error);
```

Which would call `this.guild.members.unban(this.user.id);`